node/src/chain_spec.rs

```diff
@@ -41,7 +41,7 @@ pub fn authority_keys_from_seed(s: &str) -> (AuraId, GrandpaId) {
 pub fn chain_properties() -> Properties {
 	let mut properties = Properties::new();
 	properties.insert("tokenDecimals".into(), 9.into());
-	properties.insert("tokenSymbol".into(), "ANMO".into());
+	properties.insert("tokenSymbol".into(), "ANML".into());
 	properties.insert("ss58Format".into(), 64.into());
```

Comment:
@justinFrevert plas review [this](https://www.notion.so/dot-matrix/ChainSpec-272b336fbe064820b7c064f2daf9f290) ticket

Reply:
@justinFrevert  for reference please see [here](https://github.com/hicommonwealth/edgeware-node/blob/master/node/cli/src/chain_spec.rs#L541-L544)